utilities/virt.py

```diff
@@ -1006,26 +1014,46 @@ def _sc_name_for_storage_api():
         return (
             self.data_volume.pvc.instance.spec.accessModes
             if self.data_volume
-            else self.pvc.instance.spec.accessModes
-            if self.pvc
-            else self.data_volume_template["spec"][api_name].get("accessModes")
-            or StorageProfile(name=_sc_name_for_storage_api()).instance.status["claimPropertySets"][0]["accessModes"]
+            else (
+                self.pvc.instance.spec.accessModes
+                if self.pvc
+                else self.data_volume_template["spec"][api_name].get("accessModes")
+                or StorageProfile(name=_sc_name_for_storage_api()).instance.status["claimPropertySets"][0][
+                    "accessModes"
+                ]
+            )
         )
 
     @property
     def virtctl_port_forward_cmd(self):
         return f"{VIRTCTL} port-forward --stdio=true vm/{self.name}.{self.namespace} {SSH_PORT_22}"
 
-    @property
-    def login_params(self):
-        return py_config["os_login_param"][self.os_flavor]
+    def set_login_params(self):
+        _login_params = deepcopy(py_config["os_login_param"][self.os_flavor])
```

Comment:
`py_config["os_login_param"]` is still taken from bitwarden?
https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/conftest.py#L721